### ISSUE TITLE
Error when running with python-novaclient 2.27.0

### ISSUE THREAD

**Issue by neuroid** [rackspace/pyrax#571]
Hi,

I'm getting the following exception from pyrax:

```
[snip]
  File "/home/builds/assets-deploy/env/local/lib/python2.7/site-packages/pyrax/__init__.py", line 438, in _wrapped
    return fnc(*args, **kwargs)
  File "/home/builds/assets-deploy/env/local/lib/python2.7/site-packages/pyrax/__init__.py", line 484, in auth_with_token
    connect_to_services(region=region)
  File "/home/builds/assets-deploy/env/local/lib/python2.7/site-packages/pyrax/__init__.py", line 626, in connect_to_services
    cloudservers = connect_to_cloudservers(region=region)
  File "/home/builds/assets-deploy/env/local/lib/python2.7/site-packages/pyrax/__init__.py", line 679, in connect_to_cloudservers
    extensions = cs_shell._discover_extensions("1.1")
AttributeError: 'OpenStackComputeShell' object has no attribute '_discover_extensions
```

Most likely caused by:
https://github.com/openstack/python-novaclient/commit/cb7be1281a6487692d9b7301bd19b9f4de4bda31

**Comment by jasondunsmore** [comment]
I'm seeing this as well.

**Comment by jasondunsmore** [comment]
https://github.com/rackspace/pyrax/pull/573

**Comment by briancurtin** [comment]
Fixed and released as 1.9.5 - https://pypi.python.org/pypi/pyrax/1.9.5
